ha/core/event_analyzer/parser/parser.py

```diff
@@ -145,3 +152,60 @@ def parse_event(self, msg: str) -> HealthEvent:
 
         except Exception as e:
             raise EventParserException(f"Failed to parse IEM. Message: {msg}, Error: {e}")
+
+
+class ClusterResourceParser(Parser):
+    """
+    Subscriber for event analyzer to pass msg.
+    """
+
+    def __init__(self):
+        """
+        Init method.
+        """
+        super(ClusterResourceParser, self).__init__()
+        ConfigManager.init("event_analyzer")
+        Log.info("ClusterResource Parser is initialized ...")
+
+    def parse_event(self, msg: str) -> HealthEvent:
+        """
+        Parse event.
+        Args:
+            msg (str): Msg
+        """
+        try:
+            message = json.dumps(ast.literal_eval(msg))
+            cluster_resource_alert = json.loads(message)
+            cluster_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}cluster_id")
```

Comment:
same hare, instead of fetching every time you can fetch it in __init__() check it if possible?

Reply:
This is a merge request to merge the existing code from fault_k8s branch to kubernetes branch
No code change should be done except for fixing conflicts/merge issues

Reply:
ok